Pipfile

```diff
@@ -15,6 +15,7 @@ sklearn = "*"
 papermill = "*"
 ibm-cloud-sdk-core = "*"
 ibm-watson = "*"
+flair = "*"
```

Comment:
I think you'll need to update the pipfile.lock as well for this to pass the build check. 

run `pipenv lock`